scripts/server_scripts/mock_health_event_publisher.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env python3
+# pylint: disable=import-error
+
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+
+"""Module helps in generating the mock health event and publishing it to the message bus."""
+# This script is provided by HA. If any changes are done by HA Dev team, script needs to be
+# copied from cortx-ha/ha/util/health_generator/mock_health_event_publisher.py
+
+
+import argparse
+import sys
+import pathlib
+import errno
+import json
+import time
+
+from cortx.utils.conf_store import Conf
+from cortx.utils.event_framework.health import HealthAttr, HealthEvent
+from ha.util.conf_store import ConftStoreSearch
+from ha.core.config.config_manager import ConfigManager
+from ha import const
+from ha.util.message_bus import MessageBus
+from ha.core.system_health.const import HEALTH_STATUSES
+
+DOCKER_ENV_FILE = '/.dockerenv'
+_INDEX = 'cortx'
+CLUSTER_CONFIG_FILE = '/etc/cortx/cluster.conf'
+CONFIG_FILE_FORMAT = 'yaml'
+_EVENTS_KEY = 'events'
+_SOURCE_KEY = 'source'
+_NODE_ID_KEY = 'node_id'
+_RESOURCE_TYPE_KEY = 'resource_type'
+_RESOURCE_ID_KEY = 'resource_id'
+_RESOURCE_STATUS_KEY = 'resource_status'
+_SPECIFIC_INFO_KEY = 'specific_info'
+_DELAY_KEY = 'delay'
+
+
+def is_container_env() -> bool:
+    """Returns True if environment is docker container else False."""
+    docker_env_file_path = pathlib.Path(DOCKER_ENV_FILE)
+    if docker_env_file_path.exists():
+        return True
+    return False
+
+
+# pylint: disable=redefined-outer-name
```

Comment:
Is solving these scope issue in our purview. May be same feedback should be given to HA team.

Reply:
This is not causing any behavioral changes in script.